src/engines/dropped-frames-watcher.js

```diff
@@ -29,8 +29,8 @@ class DroppedFramesWatcher extends FakeEventTarget {
       this._eventManager.listen(this._mediaSourceAdapter, CustomEventType.FPS_DROP, event => this._triggerFPSDrop(event.payload.data));
       return;
     }
-    if (this._getDroppedAndDecodedFrames()[0] === NOT_SUPPORTED) {
-      DroppedFramesWatcher._logger.debug('Dropped frame watcher is not supported on this environment');
+    if (this._getDroppedAndDecodedFrames()[0] === NOT_SUPPORTED || !this._config.capLevelOnFPSDrop) {
```

Comment:
switch order - config is more important than API support.
Also, the debug level is misleading, flag means it is disabled, not supported check should mention that it is not supported.

Reply:
changed